.gitignore

```diff
@@ -17,6 +17,7 @@
 .libs
 .project
 .settings
+*.user
```

Comment:
QtCreate files would best be excluded via local env (`~/.gitignore`).

Reply:
It is more convenient to join it

Reply:
@KangLin, we don't add ignore for non-related entries in curl specific .gitignore files.

Reply:
> 
> On 12 Nov 2018, at 10:26, Viktor Szakats ***@***.***> wrote: @vszakats requested changes on this pull request. curl has nothing to do with Qt or QtCreator, nor does it create such files as part of its build process. Why would you want to exclude parts of it from curl specifically?
> Agreed, the .gitignores in the tree should only cover files created by the build and testprocess of curl/libcurl.

OK!